scripts/zkai_claim_audit_comparison_artifacts_gate.py

```
@@ -0,0 +1,917 @@
+
+def validate_non_claim_inventory(payload: dict[str, Any]) -> None:
+    non_claims = set(non_claims_from(payload))
+    required = {
+        "not a NANOZK proof-size win",
+        "not a Jolt Atlas proof-size win",
+        "not a GKR matched d128 proof-size win",
+        "not a compact statement-binding proof-size comparison",
+    }
+    missing = sorted(required - non_claims)
+    if missing:
+        raise ClaimAuditError(f"global non-claim drift: {missing[0]}")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)